addons/io_hubs_addon/components/definitions/reflection_probe.py

```diff
@@ -759,8 +700,10 @@ def draw(self, context, layout, panel):
             sub_row = row.row(align=True)
             sub_row.enabled = False
             add_link_indicator(sub_row, self.envMapTexture)
-        row.operator("image.hubs_open_reflection_probe_envmap",
-                     text='', icon='FILE_FOLDER')
+        row.context_pointer_set("hubs_component", self)
+        row.context_pointer_set("host", self.envMapTexture)
```

Comment:
The host for this should be `context.active_object` and not the image for the reasons stated above.